### DISCUSSION TITLE
playback speed

### DISCUSSION THREAD

**Discussion by Nobbyhexer** [schreibfaul1/ESP32-audioI2S#315] [General]
I play Mp3 files from the SD card.
Is there a possibility to increase/decrease the playback speed?

**Comment by schreibfaul1** [comment]
this is not intended. You can set 'setSampleRate(uint32_t sr)' to public and then use that to change the rate. Slower is always possible, faster about up to 1.5 times the speed.

**Comment by Nobbyhexer** [comment]
Thanks for this info!

**Comment by silvan2468** [comment]
Hey schreibfaul.
Your name already says it: Schreibfaul ;)
But I would also be interested to implement a function to speed up/down the sound. Like this your library also could be used for example as sound module on remote controled cars, planes, boats,...
What do you think?
Thanks and have a nice weekend
Silvan
